aws/PAS-network-environment-NAT.json

```diff
@@ -1,1655 +1,2102 @@
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "CPMMainSubnet"
+        }
+      }
+    },
+    "CPMDRRTAssociation": {
+      "Type": "AWS::EC2::SubnetRouteTableAssociation",
+      "Properties": {
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "CPMDRSubnet"
+        }
+      }
+    },
+    "PTAMainRTAssociation": {
+      "Type": "AWS::EC2::SubnetRouteTableAssociation",
+      "Properties": {
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "PTAMainSubnet"
+        }
+      }
+    },
+    "PTADRRTAssociation": {
+      "Type": "AWS::EC2::SubnetRouteTableAssociation",
+      "Properties": {
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "PTADRSubnet"
+        }
+      }
+    },
+    "PSMMainRTAssociation": {
+      "Type": "AWS::EC2::SubnetRouteTableAssociation",
+      "Properties": {
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "PSMMainSubnet"
+        }
+      }
+    },
+    "PSMSecondaryRTAssociation": {
+      "Type": "AWS::EC2::SubnetRouteTableAssociation",
+      "Properties": {
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "PSMSecondarySubnet"
+        }
+      }
+    },
+    "PSMPMainRTAssociation": {
+      "Type": "AWS::EC2::SubnetRouteTableAssociation",
+      "Properties": {
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "PSMSSHMainSubnet"
+        }
+      }
+    },
+    "PSMPSecondaryRTAssociation": {
+      "Type": "AWS::EC2::SubnetRouteTableAssociation",
+      "Properties": {
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        },
+        "SubnetId": {
+          "Ref": "PSMSSHSecondarySubnet"
+        }
+      }
+    },
+    "PASPrivateNATRoute": {
+      "Type": "AWS::EC2::Route",
+      "Properties": {
+        "DestinationCidrBlock": "0.0.0.0/0",
+        "NatGatewayId": {
+          "Ref": "PASNATGW"
+        },
+        "RouteTableId": {
+          "Ref": "PASPrivateRT"
+        }
+      }
+    },
+    "PASPublicNATRoute": {
+      "Type": "AWS::EC2::Route",
+      "Properties": {
+        "DestinationCidrBlock": "0.0.0.0/0",
+        "RouteTableId": {
+          "Ref": "PASPublicRT"
+        },
+        "GatewayId": {
+          "Ref": "PASIGW"
+        }
+      },
+      "DependsOn": "PASGWAttachment"
+    },
+    "VaultSGIngress1": {
+      "Type": "AWS::EC2::SecurityGroupIngress",
+      "Properties": {
+        "GroupId": {
+          "Ref": "VaultSG"
+        },
+        "IpProtocol": "tcp",
+        "FromPort": 1858,
+        "ToPort": 1858,
+        "SourceSecurityGroupId": {
+          "Ref": "VaultSG"
+        },
+        "Description": "Vault to Vault connection"
```

Comment:
Vault to Vault?
Did you mean "Casos Clients to Vault?"

Reply:
Vault to Vault DR